<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>656</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Алгоритмы построения окружностей и эллипсов</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>341</width>
      <height>501</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0" colspan="2">
      <widget class="QComboBox" name="algorithm">
       <item>
        <property name="text">
         <string>Брезенхем</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Средняя точка</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Каноническое уравнение</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Параметрическое уравнение</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Алгоритм из библиотеки</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QSpinBox" name="spinBox_cx">
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="value">
        <number>500</number>
       </property>
      </widget>
     </item>
     <item row="7" column="0" rowspan="2" colspan="2">
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="18" column="0" colspan="2">
      <spacer name="verticalSpacer_8">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="12" column="1">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>B</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>R</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Выбор алгоритма</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QSpinBox" name="spinBox_a">
       <property name="maximum">
        <number>650</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="23" column="0" colspan="2">
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="14" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Цвет</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QSpinBox" name="spinBox_r">
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="17" column="0" colspan="2">
      <widget class="Line" name="line_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <spacer name="verticalSpacer_6">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="9" column="0">
      <widget class="QRadioButton" name="circle">
       <property name="text">
        <string>Окружность</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="19" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Шаг</string>
       </property>
      </widget>
     </item>
     <item row="25" column="0" colspan="2">
      <widget class="QPushButton" name="pushButton_clear">
       <property name="text">
        <string>Очистить холст</string>
       </property>
      </widget>
     </item>
     <item row="20" column="1">
      <widget class="QSpinBox" name="spinBox_n">
       <property name="maximum">
        <number>650</number>
       </property>
       <property name="value">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item row="15" column="0">
      <widget class="QComboBox" name="color">
       <item>
        <property name="text">
         <string>Красный</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Зеленый</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Синий</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Цвет фона</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>A</string>
       </property>
      </widget>
     </item>
     <item row="22" column="0" colspan="2">
      <spacer name="verticalSpacer_7">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Центр</string>
       </property>
      </widget>
     </item>
     <item row="20" column="0">
      <widget class="QSpinBox" name="spinBox_h">
       <property name="maximum">
        <number>650</number>
       </property>
       <property name="value">
        <number>20</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="16" column="0" colspan="2">
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="19" column="1">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Количество</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QSpinBox" name="spinBox_cy">
       <property name="maximum">
        <number>650</number>
       </property>
       <property name="value">
        <number>325</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <spacer name="verticalSpacer_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="15" column="1">
      <widget class="QPushButton" name="pushButton_draw">
       <property name="text">
        <string>Построить</string>
       </property>
      </widget>
     </item>
     <item row="13" column="1">
      <widget class="QSpinBox" name="spinBox_b">
       <property name="maximum">
        <number>650</number>
       </property>
       <property name="value">
        <number>50</number>
       </property>
      </widget>
     </item>
     <item row="24" column="0" colspan="2">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="9" column="1">
      <widget class="QRadioButton" name="ellipse">
       <property name="text">
        <string>Эллипс</string>
       </property>
      </widget>
     </item>
     <item row="21" column="0" colspan="2">
      <widget class="QPushButton" name="pushButton_spectrum">
       <property name="text">
        <string>Построить спектр</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
